hazelcast/src/main/java/com/hazelcast/internal/diagnostics/DiagnosticsLogWriter.java

```diff
@@ -120,7 +120,7 @@ protected void appendDateTime() {
     private void appendDate() {
         write(calendar.get(DAY_OF_MONTH));
         write('-');
-        write(calendar.get(MONTH));
+        write(calendar.get(MONTH + 1));
```

Comment:
This adds one to the `MONTH` constant and will get the `WEEK_OF_YEAR` field instead.

Correct code is `write(calendar.get(MONTH) + 1);`